.gitignore

```diff
@@ -41,3 +41,5 @@ yarn-error.log*
 .idea
 *.sublime-*
 .vscode
+
+src/typechain
```

Comment:
- [ ] need to remove the existing files from repo

Reply:
already did

Reply:
nice ser don't know how I missed that